src/Constants.js

```diff
@@ -94,26 +94,26 @@ exports.LastChanceOptions = {
 };
 
 /**
- * The giveaways manager options
+ * The giveaways manager options.
  * @typedef GiveawaysManagerOptions
  *
  * @property {string} [storage='./giveaways.json'] The storage path for the giveaways.
  * @property {number} [updateCountdownEvery=5000] The giveaway update interval (in ms).
  * @property {number} [endedGiveawaysLifetime=null] The time (in ms) after which a ended giveaway should get deleted from the DB. ⚠ Giveaways deleted from the DB cannot get rerolled anymore!
- * @property {boolean} [hasGuildMembersIntent=false] Whether the client instance has access to the GUILD_MEMBERS intent. If set to true, everything will be faster.
+ * @property {boolean} [hasGuildMembersIntent=false] Whether the client instance has access to the "GUILD_MEMBERS" intent. If set to true, everything will be faster.
  * @property {GiveawayStartOptions} [default] The default options for new giveaways.
- * @property {Boolean} [default.botsCanWin=false] Whether the bots are able to win a giveaway.
+ * @property {Boolean} [default.botsCanWin=false] If bots can win giveaways.
  * @property {Discord.PermissionResolvable[]} [default.exemptPermissions=[]] Members with any of these permissions won't be able to win a giveaway.
  * @property {Function} [default.exemptMembers] Function to filter members. If true is returned, the member won't be able to win a giveaway.
- * @property {Discord.ColorResolvable} [default.embedColor='#FF0000'] The giveaways embed color when they are running
- * @property {Discord.ColorResolvable} [default.embedColorEnd='#000000'] The giveaways embed color when they are ended
- * @property {Discord.EmojiIdentifierResolvable} [default.reaction='🎉'] The reaction to participate in the giveaways
- * @property {LastChanceOptions} [default.lastChance] The last chance system parameters
+ * @property {Discord.ColorResolvable} [default.embedColor='#FF0000'] The color of the giveaway embed when it is running.
+ * @property {Discord.ColorResolvable} [default.embedColorEnd='#000000'] The color of the giveaway embed when it has ended.
+ * @property {Discord.EmojiIdentifierResolvable} [default.reaction='🎉'] The reaction to participate in the giveaway.
```

Comment:
```suggestion
 * @property {Discord.ColorResolvable} [default.embedColor='#FF0000'] The color of the giveaway embeds when they are running.
 * @property {Discord.ColorResolvable} [default.embedColorEnd='#000000'] The color of the giveaway embeds when they have ended.
 * @property {Discord.EmojiIdentifierResolvable} [default.reaction='🎉'] The reaction to participate in giveaways.
```
Since this if for the manager options, it should stay in plural because it refers to all giveaways